graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java

```diff
@@ -163,6 +161,26 @@ public GraphQLSchema build() {
     return builder.build();
   }
 
+  private void addAtomicDirective(GraphQLSchema.Builder builder) {
+    builder.additionalDirective(
+        GraphQLDirective.newDirective()
+            .validLocation(Introspection.DirectiveLocation.MUTATION)
+            .name(SchemaConstants.ATOMIC_DIRECTIVE)
+            .description("Instructs the server to apply the mutations in a LOGGED batch")
+            .build());
+  }
+
+  private void addAsyncDirective(GraphQLSchema.Builder builder) {
+    builder.additionalDirective(
+        GraphQLDirective.newDirective()
+            .validLocation(Introspection.DirectiveLocation.MUTATION)
+            .name(ASYNC_DIRECTIVE)
+            .description(
+                "Instructs the server to apply the mutations in an Async way. "
```

Comment:
minor: `... apply the mutations in an Async way.` --> `... apply the mutations asynchronously without waiting for the result` ?